hotel/tests/test_attendee_properties.py

```diff
@@ -0,0 +1,25 @@
+from hotel import *
+import pytest
+from sideboard.tests import patch_session
+
+
+@pytest.fixture(scope='session', autouse=True)
+def init_db(request):
+    patch_session(Session, request)
+    initialize_db()
+
+
```

Comment:
These were the only two tests from the core repo which needed to be moved here.  As always, it would be good to have more tests than we actually have at the moment.  Someday.  Maybe.  Or never.